prow/manifests/overlays/metal3/config.yaml

```diff
@@ -1843,6 +1975,39 @@ presubmits:
       agent: jenkins
       always_run: false
       optional: true
+    # name: {job_prefix}-{image_os}-dev-env-integration-test-{capm3_target_branch}
```

Comment:
Lets not add tests for metal3-io/metal3-docs repo and

Reply:
I think the preview cuts the code in a poorly informative way. The test are actually under metal3-io/baremetal-operater, metal3-io/cluster-api-provider-metal3, metal3-io/meta3-dev-env, metal3-io/project-infra and metal3-io/ip-address-manager. 

I checked that these are the projects where integration tests are run. Are these all correct and/or I'm I missing some?